### PR TITLE
Update func s2i task

### PR DESCRIPTION
**Pull request by lance**
Updates the `REGISTRY` parameter to have a meaningless default value, and improves the pipeline name
/assign @jrangelramos

### CODE REVIEW

File: openshift/overrides/pkg/pipelines/resources/tekton/pipeline/dev-console/0.1/pipeline.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: tekton.dev/v1beta1
 kind: Pipeline
 metadata:
-  name: devconsole-function-pipeline
+  name: devconsole-nodejs-function-pipeline
```
Comment (jrangelramos) [inline openshift/overrides/pkg/pipelines/resources/tekton/pipeline/dev-console/0.1/pipeline.yaml:4]: will the pipeline be specific for nodejs only?  if so perhaps the filename could be `pipeline-nodejs.yaml` so we could have in future a `pipeline-quarkus.yaml` and so on...

But the yaml seems to me generic and could be used in future for other runtimes..

File: openshift/patches/0006-tekton-pipelines.patch [hunk 2]
```diff
@@ -32,3 +32,21 @@ index bf90adfa..4e655633 100644
...
+       description: The registry associated with the function image.
++      default: ""
+     - name: PATH_CONTEXT
+       description: The location of the path to run s2i from.
+       default: .
+@@ -114,6 +115,9 @@ spec:
+           $(params.IMAGE) docker://$(params.IMAGE)
+
+         cat $(workspaces.source.path)/image-digest | tee /tekton/results/IMAGE_DIGEST
++        echo ""
```
Comment (matejvasek) [inline openshift/patches/0006-tekton-pipelines.patch:47]: what is this for?

Comment (matejvasek) [reply openshift/patches/0006-tekton-pipelines.patch:47]: isn't that just debugging?

Comment (lance) [reply openshift/patches/0006-tekton-pipelines.patch:47]: That file is used by the pipeline, iiuc, here: https://github.com/openshift-knative/kn-plugin-func/blob/a80fa69fa32b6c42575b59629332c74ede8f77c0/openshift/overrides/pkg/pipelines/resources/tekton/pipeline/dev-console/0.1/nodejs-pipeline.yaml#L82

Comment (matejvasek) [reply openshift/patches/0006-tekton-pipelines.patch:47]: I do understand `cat $(workspaces.source.path)/image-digest | tee /tekton/results/IMAGE_DIGEST` but what about the three lines below?

Comment (matejvasek) [reply openshift/patches/0006-tekton-pipelines.patch:47]: iiuic `cat $(workspaces.source.path)/image-digest | tee /tekton/results/IMAGE_DIGEST` also prints the hash, so `cat $(workspaces.source.path)/image-digest` seems redundant, am I wrong?

Comment (lance) [reply openshift/patches/0006-tekton-pipelines.patch:47]: No, you are not wrong. https://github.com/openshift-knative/kn-plugin-func/pull/573

**Pull request merged**